### ISSUE TITLE
Very low contrast errors thrown for text-muted and badge styles

### ISSUE THREAD

**Issue by nubz** [CJSCommonPlatform/ngGovUk#24]
Using the WAVE accessibility plugin for Chrome we get very low contrast errors for the .text-muted text colour (#DEE0E2 on white) and .badge colours (white text on #DEE0E2) - in fact without WAVE we can see they are barely legible. My thoughts on the project I'm working on is that we don't use text-muted at all and I'll be seeking User Research guidance on removing it anyway and also on not using default badges - seems to be a bootstrap only thing and not a gov.uk thing! But where it is used it will fail accessibility assessments.

**Comment by nubz** [comment]
Further to this I have found .breadcrumb > .active also using the same very low contrast colour rendering the active page in the breadcrumb trail illegible.

**Comment by vygis** [comment]
`breadcrumb > .active` has been fixed. The low contrast on `.text-muted` and `.badge` were caused by overriding Bootstrap's greyscale palette variables. I've updated these to have more contrast (and also some tweaks to `badge` - might as well have it properly styled).
